managed/services/dbaas/dbaas_client.go

```diff
@@ -214,10 +215,70 @@ func (c *Client) GetLogs(ctx context.Context, in *controllerv1beta1.GetLogsReque
 }
 
 // GetResources returns all and available resources of a Kubernetes cluster.
-func (c *Client) GetResources(ctx context.Context, in *controllerv1beta1.GetResourcesRequest, opts ...grpc.CallOption) (*controllerv1beta1.GetResourcesResponse, error) {
+func (c *Client) GetResources(ctx context.Context, in *controllerv1beta1.GetResourcesRequest, _ ...grpc.CallOption) (*controllerv1beta1.GetResourcesResponse, error) {
 	c.connM.RLock()
 	defer c.connM.RUnlock()
-	return c.kubernetesClient.GetResources(ctx, in, opts...)
+	kClient, err := kubernetes.New(in.KubeAuth.Kubeconfig)
+	if err != nil {
```

Comment:
Thanks. It looks good so far. Can we move it to `managed/services/management/dbaas` and get rid of GetResources function in dbaas_client? We will drop it soon and there's no point in moving it here. Also, it would be great to add some tests here. What do you think?

Reply:
I agree